.github/workflows/build.yml

```diff
@@ -0,0 +1,51 @@
+name: lf-sample-repository-api-nodejs-CI
+
```

Comment:
Comparing with AC Team's TS project, it seems the convention is to name this file `main.yml`.